### DISCUSSION TITLE
NFT PAGE ideas

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#74] [Ideas]
I believe there is potential for further development to increase the purchasing of Bitcanna NFTs. It would be interesting to obtain the floor price and check if there are any NFTs from BitCanna Buddhead available below that price, which could present an opportunity to make a purchase. From a user experience standpoint, it is not user-friendly to go through multiple steps such as navigating to Stargaze, connecting your wallet, accessing the marketplace, searching for the collection, and selecting "buy." These extra clicks could be eliminated by having a dedicated page that showcases the best deals available, if any. If an NFT is not available below the floor price, the page could display the top ten prices for purchasing. Clicking on the "buy" option could directly redirect users to the Stargaze website or even enable direct purchases from the Bitcanna Weedwallet, which would be an impressive feature.

Additional Benefit 1: Users can bookmark this dedicated page in their browser and easily access it whenever they want to check for good deals on the BitCanna Buddhead Collection. It reduces the effort required to stay updated with potential opportunities, as they can do so with just one click.

We could also introduce rewards for NFT holders based on the amount of time they have held their NFTs and the number of airdrops they have received. As users accumulate more time and airdrops, they can progress through different levels on their wallet, such as Newbud, Veterweed, Weedelite, etc. This system encourages users to hold onto their NFTs and provides a sense of growth and achievement.

Additional Benefit 2: Users will have the ability to capture screenshots of their rewarding progress within Bitcanna, which can be shared on social media. This not only promotes Bitcanna but also enhances its image and reputation among users.

**Comment by TJ-BitCanna** [comment]
Thanks for the ideas!

This will not be covered in the first release, but certainly something to explore in a future release.
